docs/plans/2026-02-15-pr-skills-design.md

```
@@ -0,0 +1,141 @@
+
+### Finish Phase (`/feature-pr finish`)
+
+1. Verify NOT on develop/main (abort if so)
+2. Run tests — must pass before proceeding
+3. Review all changes (staged + unstaged + untracked)
+4. Create meaningful commits in appropriate units (conventional commits, Korean)
+5. Push branch to remote
+6. Create squash PR to develop:
+   - **Title**: Inferred from branch prefix + Linear ID + brief description
+   - **Body** (follows PR template):
+     ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)